2019-08-01 23:35:55.250891: COMBO  :  ['Traceback (most recent call last):\n', '  File "/anaconda3/lib/python3.6/site-packages/mysql/connector/connection_cext.py", line 395, in cmd_query\n    raw_as_string=raw_as_string)\n', "_mysql_connector.MySQLInterfaceError: Duplicate entry 'ww0lLrY' for key 'PRIMARY'\n", '\nDuring handling of the above exception, another exception occurred:\n\n', 'Traceback (most recent call last):\n', '  File "/Users/huyanh/Documents/websim_dev/websim/common/utils.py", line 162, in db_insert_combo\n    cursor.execute(query, values)\n', '  File "/anaconda3/lib/python3.6/site-packages/mysql/connector/cursor_cext.py", line 266, in execute\n    raw_as_string=self._raw_as_string)\n', '  File "/anaconda3/lib/python3.6/site-packages/mysql/connector/connection_cext.py", line 398, in cmd_query\n    sqlstate=exc.sqlstate)\n', "mysql.connector.errors.IntegrityError: 1062 (23000): Duplicate entry 'ww0lLrY' for key 'PRIMARY'\n"]
